libs/context/keys.go

```diff
@@ -154,6 +154,11 @@ const (
 	AWSRegionCTXKey CTXKey = "aws_region"
 	// RequireUpholdCountryCTXKey - the context key for getting the rate limit
 	RequireUpholdCountryCTXKey CTXKey = "require_uphold_country"
+
+	// WalletGeolocationDisabledBucketCTXKey the context key for getting the S3 bucket for wallet creation disabled geolocations.
+	WalletGeolocationDisabledBucketCTXKey CTXKey = "disabled-wallet-geolocations-bucket"
```

Comment:
Can we use the same bucket that the parameters service uses, `ParametersMergeBucketCTXKey`?